lib/shared/bucketing-assembly-script/index.cjs

```diff
@@ -0,0 +1,16 @@
+const { instantiate } = require('./build/bucketing-lib.debug.commonjs')
+
+const url = new URL("build/bucketing-lib.release.commonjs.wasm", import.meta.url)
+
+exports.instantiate = async () => {
```

Comment:
can you add some docs of what this is trying to do?

Reply:
I dont really know what its trying to do exactly, i just copied it from the ESM output. I can comment my best guess though